### ISSUE TITLE
Implement support for other line styles

### ISSUE THREAD

**Issue by mum4k** [mum4k/termdash#81]
Currently only one line style is supported:

https://github.com/mum4k/termdash/blob/master/draw/line_style.go

**Comment by infinytum** [comment]
I've implemented a LineStyleDouble which looks like this:

<img width="1662" alt="image" src="https://user-images.githubusercontent.com/43315617/51633224-73601700-1f51-11e9-8150-ed90feeb3ce6.png">

**Comment by infinytum** [comment]
I'll open a pull-request as soon as I figured out how to not submit my commits from my other pull-request too...

**Comment by mum4k** [comment]
To send this as a separate PR, I recommend creating a new fork off the devel branch, making the changes and sending a PR against it.

Let me know if you would like to get more details on this.

**Comment by infinytum** [comment]
I cant think of any made of ASCII characters (https://theasciicode.com.ar)

If we can use Unicode, I can think of one or two more... I'd have to test that out first though

**Comment by mum4k** [comment]
Looks great, thanks for adding both. We indeed can use unicode, but I think you already know that ;)

I will leave it up to you to close this issue, not sure if you are still working on other line styles.

**Comment by infinytum** [comment]
Thank you! Currently I am out of ideas for new borders. I will file a pull request if I get more ideas.

**Comment by mum4k** [comment]
Sounds good, let's close this for now, we can as well follow the YAGNI principle and add it only once we need it ;)
